### DISCUSSION TITLE
What is the difference between direct, CDN, relay and fake mode? تفاوت حالت‌های دایرکت، سی‌دی‌ان، ریلای و فیک تو تنظیمات دامنه چیه؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#118] [Q&A سوالات رایج] [answered]
تفاوت حالت‌های دایرکت، سی‌دی‌ان و ریلای تو تنظیمات دامنه چیه؟

**Comment by babveihi** [comment]
حالت fake چیه توی پانل ؟

**Comment by hiddify-com** [comment, marked as answer]
حالت CDN:
برای زمانی که tls توسط یک سرور میانی مثل CDN ترمینیت میشه در این حالت فقط WebSocket کار میکنه و سایر سرویس ها نمایش داده نمیشود

حالت Relay:
در زمانی که بیش از یک آی پی یا IPv6 روی سرور وجود دارد باید برای آن آی پی ها از این گزینه استفاده شود
همچنین زمانی که یک تونل tcp بین سرویس ایران و خارج ایجاد کرده باشید مفید خواهد بود.

حالت Fake:
برای مثال میخواهید سایت soft98.ir را در هدر به عنوان سایت خود قرار دهید تا فیلترچی فکر کند شما دارید soft98 را میبینید البته به نظر ما این کار باعث فیلتر شدن سریع تر میشود

**Reply by babveihi** [reply]
خب الان برای اضافه کردن آی پی 6 اجازه نمیده با همون نام دامنه که ساب دامین ساختیم و رجیستر کردیم دوباره دامنه رو توی پانل ثبت کنیم
 ، ضمنا اصلا آیتمی هم وجود نداره که آی پی 6 رو ازمون بخواد 
من یه ساب دامنه ثبت کردم تو کلودفلر و هم آی پی 4 و هم آی پی 6 رو براش گذاشتم
ضمنا برای هر دو هم پراکسی روشن هست تو کلودفلر
